addons/actions/src/containers/ActionLogger/index.tsx

```diff
@@ -63,13 +83,8 @@ export default class ActionLogger extends React.Component {
   }
 }
 
-ActionLogger.propTypes = {
+(ActionLogger as any).propTypes = {
```

Comment:
Is this necessary?

Reply:
Actually do we still need propTypes?

Reply:
👍  I'm more from Angular world so I read some blog posts about propTypes and TS and indeed it looks like `propTypes` is not really necessary. So I added a last commit removing all references to propTypes in this addon.